src/client/Client.cc

```diff
@@ -9714,25 +9741,25 @@ int Client::_link(Inode *in, Inode *dir, const char *newname, int uid, int gid,
   return res;
 }
 
-int Client::ll_link(Inode *parent, Inode *newparent, const char *newname,
+int Client::ll_link(Inode *in, Inode *newparent, const char *newname,
```

Comment:
Why the name change, and did it need to go in the giant patch instead of in a separate one for easier review?